partiql-planner/src/main/kotlin/org/partiql/planner/internal/transforms/PlanTransform.kt

```diff
@@ -242,6 +242,14 @@ internal class PlanTransform(private val flags: Set<PlannerFlag>) {
             return operators.aggregate(input, calls, groups)
         }
 
+        override fun visitRelOpWith(node: Rel.Op.With, ctx: PType): Any {
```

Comment:
Why return `Any` and not `org.partiql.plan.rel.Rel`?

Reply:
In the end, it doesn't matter. These all get cast to Rels.